vkcs/images_image.go

```diff
@@ -165,11 +167,19 @@ func resourceImagesImageExpandProperties(v map[string]interface{}) map[string]st
 			properties[key] = v
 		}
 	}
-	properties["store"] = "s3"
 
 	return properties
 }
 
+func resourceImagesImageNeedsDefaultStore(endpoint string) bool {
```

Comment:
We should check that domain contains mask only in the host part of endpoint URL

Reply:
Rewrote to check only the host part